tutorials/plot_visualize_raw.py

```diff
@@ -70,11 +70,12 @@
 # positions available. The channels are grouped by sensor positions to 8 evenly
 # sized regions. Notice that for this to work effectively, all the data
 # channels in the channel array must be present. The ``order`` parameter allows
-# to use a customize the order (picks). Here we use the butterfly mode which
-# groups the channels the same way as the position mode. To toggle between
-# these modes, press 'b' key when the plotter window is active. Notice that
-# ``group_by`` also effects the channel groupings in butterfly mode.
-raw.plot(group_by='butterfly')
+# to customize the order and select a subset of channels for plotting (picks).
+# Here we use the butterfly mode and group the channels by position. To toggle
+# between regular and butterfly modes, press 'b' key when the plotter window is
+# active. Notice that ``group_by`` also affects the channel groupings in
+# butterfly mode.
+raw.plot(butterfly=True)
```

Comment:
according to the text above there should be a `group_by='position'` here, right?